assets/src/stories-editor/plugins/story-settings-panel.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { metaToAttributeNames } from '../helpers';
+
+/**
+ * WordPress dependencies
+ */
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+import { compose } from '@wordpress/compose';
+
+import {
+	withSelect,
+	withDispatch,
+} from '@wordpress/data';
+
+import {
+	SelectControl,
+	RangeControl,
+} from '@wordpress/components';
+
+let MetaFields = ( props ) => {
```

Comment:
Instead of using `let` I'd name a "dumb" component like this "MetaFields" and then the enhanced one "EnhancedMetaFields` or so.